### PR TITLE
QA-226: Use new release_tool CLI option to run feature branches pipeline

### PR DESCRIPTION
**Pull request by lluiscampos**
Depends on https://github.com/mendersoftware/integration/pull/1066

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -46,7 +46,11 @@ func repoToBuildParameter(repo string) string {
 
 // Use python script in order to determine which integration branches to test with
 func getIntegrationVersionsUsingMicroservice(repo, version string, conf *config) ([]string, error) {
-	c := exec.Command("release_tool.py", "--integration-versions-including", repo, "--version", version)
+	cmdArgs := []string{"--integration-versions-including", repo, "--version", version}
+	if strings.HasPrefix(version, "feature-") {
```
Comment (merlin-northern) [inline helpers.go:50]: can we define the prefix "feature-" as a const?

**Review by merlin-northern — APPROVED** [review]
small nitpick looks good to me

File: helpers.go (hunk 1 above)
Comment (lluiscampos) [reply helpers.go:50]: Sure, coming up.

**Review by merlin-northern — APPROVED** [review]
lgtm

**Review by kacf — APPROVED** [review]

**Pull request merged**